exporter/datadogexporter/example/config.yaml

```
@@ -148,6 +148,12 @@ exporters:
       #   instrumentation:express.server: express
       #   go.opentelemetry.io_contrib_instrumentation_net_http_otelhttp.client: http.client
 
+      ## @param span_name_as_resource_name - use OpenTelemetry semantic convention for span naming - optional
+      ## Option created to maintain similarity with the OpenTelemetry semantic conventions as discussed in the issue below.
+      ## https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/trace/semantic_conventions
+      ## https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/1909
+      # span_name_as_resource_name: true
```

just checking here that the example set to `true` was intentional since the default is `false`. I see some of the other options (endpoint) appears to be showing the default. 

Nit: please add line to separate the option and the documentation as for the other options.

```suggestion
      #
      # span_name_as_resource_name: true
```

---

Well, I thought on doing like this because I wanted to give an example on how to use the feature, if you don't want to use it there is no need to set the configuration.